src/google/protobuf/util/json_util.cc

```diff
@@ -50,7 +50,8 @@ namespace util {
 
 namespace internal {
 ZeroCopyStreamByteSink::~ZeroCopyStreamByteSink() {
-  stream_->BackUp(buffer_size_);
+  if (buffer_size_ > 0)
```

Comment:
Could you put in curly braces around the body of the if statement, i.e.

if (buffer_size_ > 0 ) {
  ...
}

We usually go with that convention just to be safe.